### DISCUSSION TITLE
Detection remains even though object has gone

### DISCUSSION THREAD

**Discussion by djskinner** [blakeblackshear/frigate#2987] [Ask A Question] [answered]
Since the last major release I frequently experience a problem whereby the system thinks an object is still there when it has completely left the field of view. It seems to occur around the boundary of zones. This causes constant person detection notifications until a new person comes along to "reset" it.

![download](https://user-images.githubusercontent.com/2514172/159113813-fef94928-9ea5-4308-9fde-412bf19db1e6.png)

The screenshot above is a typical example. A person entering/leaving the zone event is triggered indefinitely. Unless another person comes along I have to restart Frigate to reset it or home assistant sends me a notification every minute that a person has entered the zone.

Any ideas what settings I need to tweak to get back to the expected behaviour?

**Comment by NickM-27** [comment, marked as answer]
Hi, please see the conversation here https://github.com/blakeblackshear/frigate/issues/2965

If the circumstances aren't at all similar I'll need some additional details

**Reply by djskinner** [reply]
Thanks. Yeah I think that is sufficiently similar to the issue I'm having. Will follow along with #2965
